src/views/BottomTabBar.js

```diff
@@ -168,7 +168,28 @@ class TabBarBottom extends React.Component<Props> {
 
   _shouldUseHorizontalLabels = () => {
     const { routes } = this.props.navigation.state;
-    const { isLandscape, dimensions, adaptive, tabStyle } = this.props;
+    const {
+      isLandscape,
+      dimensions,
+      adaptive,
+      tabStyle,
+      tabDirection,
+    } = this.props;
+
+    if (tabDirection) {
+      let direction;
+      if (typeof direction === 'string') {
+        direction = tabDirection;
+      } else {
+        direction = tabDirection({
```

Comment:
this else clause will only be entered when `tabDirection` is falsey, and then we try to call `tabDirection` as a function which will error

Reply:
we're checking `tabDirection` above